apps/desktop/src/app/main-layout/topbar.component.ts

```diff
@@ -1,16 +1,46 @@
 import { ChangeDetectionStrategy, Component } from '@angular/core';
 import { Router } from '@angular/router';
+import { SDKService, UserService } from '@flaps/auth';
+import { map, of, switchMap } from 'rxjs';
+import { stfAnimations } from '@flaps/pastanaga';
+import { SyncService } from '../sync/sync.service';
+
 
 @Component({
   selector: 'da-topbar',
   templateUrl: 'topbar.component.html',
   styleUrls: ['./topbar.component.scss'],
   changeDetection: ChangeDetectionStrategy.OnPush,
+  animations: [stfAnimations],
 })
 export class TopbarComponent {
-  constructor(private router: Router) {}
+  constructor(
+    private router: Router,
+    private user: UserService,
+    private sync: SyncService,
+    private sdk: SDKService,
+  ) {}
+
+  menuOpen = false;
+
+  account = of(this.sync.getAccount()).pipe(
+    switchMap((account) => this.sdk.setCurrentAccount(account)),
```

Comment:
Why do we need to do that here?
It is already done in the SyncService constructor.
(I mean, maybe we need to repeat the `setCurrentAccount` for some reason, but if so, that's fine, even though it is a bit weird. Maybe at some point we will need a `getCurrentAccount` in SDKService)

Reply:
fixed